src/frontend/apps/lti_site/components/VideoWidgetProvider/widgets/DownloadVideo/index.spec.tsx

```diff
@@ -123,4 +128,129 @@ describe('<InstructorDownloadVideo />', () => {
     const downloadButton = screen.getByRole('button', { name: 'Download' });
     expect(downloadButton).toBeDisabled();
   });
+
+  it('check toggle allow download disable attribut', async () => {
+    const mockedVideo = videoMockFactory();
+    const allowDownloadToggleLabel = 'Allow video download';
+    const allowDownloadToggleFail = 'Update failed, try again.';
+
+    render(
+      wrapInVideo(
+        <InfoWidgetModalProvider value={null}>
+          <DownloadVideo />
+        </InfoWidgetModalProvider>,
+        mockedVideo,
+      ),
+    );
+
+    screen.getByText(allowDownloadToggleLabel);
```

Comment:
```suggestion
    const allowDownloadCheckbox = screen.getByRole('checkbox', { name: allowDownloadToggleLabel });
```

Using `ByRole` is more accurate and you can use it all along the test.
This suggestion can be applied to all other tests.